### PR TITLE
W-012

### PR DESCRIPTION
**Pull request by WOJTASK1**

### CODE REVIEW

File: frontend/src/components/navbar/Navbar.css [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+  }
+ .navibutton.hovered{
+    background-color: var(--button-gray);
+    color:var(--main-color)!important;
+    text-decoration: none;
+  }
+  #btn{
+    position: absolute;
+    left:5em;
+    background-color: var(--main-color);
```
Comment (wojciechwelfle) [inline frontend/src/components/navbar/Navbar.css:38]: delete 38 line

**Review by wojciechwelfle — CHANGES_REQUESTED** [review]
format all files and delete comments

File: frontend/src/components/navbar/Navbar.css (hunk 1 above)
Comment (WOJTASK1) [reply frontend/src/components/navbar/Navbar.css]: Already done!

**Review by wojciechwelfle — APPROVED** [review]
lgtm

**Pull request merged**